src/sonic-ctrmgrd/ctrmgr/container

```diff
@@ -0,0 +1,367 @@
+#!/usr/bin/python3 -u
+
+import argparse
+import docker
+import inspect
+import json
+import os
+import sys
+import syslog
+import time
+
+from swsscommon import swsscommon
+
+sys.path.append('/usr/share/sonic/scripts/')
```

Comment:
why do we need this, if why not install container_state in proper python library?

Reply:
I need this in a path that is accessible from inside container.
I prefer to keep a single copy in use by anyone who needs this.
Technically, no one else needs it. But when a container is killed as it does not have a chance to call container_state to mark down, the kill action calls it.

Reply:
can you remaind me why not install the ctrmgr package inside the sonic-config-engine docker? in this case you will have the library as well.

Reply:
First some points & then reasoning:
Points:
- This package is heavily used by host for remote container management.
- ctrmgrd watches the changes done by container_state in STATE-DB to act on
- container commands (ones that replace docker start/stop/kill/wait) also relies on changes done by container_state
- container_state is now *only* in host, mounted to the containers at runtime and ***all*** containers (irrespective of versions deployed) call the ***same*** script to mark up/down.

As container_state action is heavily bound at host, we would like to keep it completely under Base-OS control. Hence not installed in individual containers, which otherwise has the potential to bring in multiple versions of container_state, and complicate for no *mentionable* gain.